ansible/roles/create-api-deployment-pre-reqs/templates/terraform/iam.tf

```diff
@@ -252,6 +252,7 @@ data "aws_iam_policy_document" "deploy-user" {
 
     resources = concat(
       [local.ecs_cluster.arn],
+      [for ns in local.service_namespaces : "arn:aws:ecs:${local.region}:${local.account_id}:service/${ns}"],
```

Comment:
I can't see how this ARN uniquely identifies an ECS service. For example, how would AWS know whether `arn:aws:ecs:eu-west-2:958002497996:service/eps-coordinator` is referring to the internal-dev service or the int service? I'm not au fait with the old ARN format or the recent ARN format changes though so maybe it's fine?

Reply:
Based on some Stack Overflow searching, it sounds like even with this extra permission added, we'd just hit a second error:

> Long arn format must be used for tagging operations

I think the solution may be to delete and recreate old ECS services instead.